Package

```diff
@@ -8,7 +8,7 @@ plugin_install_dirname=ModbusC
 #   1. Generic packages we depend on in all architectures and package managers
 #   2. Architecture specific packages we depend on
 #   3. Package manager specific packages we depend on
-requirements="foglamp"
+requirements="foglamp,epel-release,libmodbus,libmodbus-devel"
```

Comment:
Please new deps removed it from here and move here `requirements="${requirements},epel-release,libmodbus,libmodbus-devel ` in this order